internal/component/prometheus/scrape/scrape.go

```diff
@@ -69,6 +69,8 @@ type Arguments struct {
 	Params url.Values `alloy:"params,attr,optional"`
 	// Whether to scrape a classic histogram that is also exposed as a native histogram.
 	ScrapeClassicHistograms bool `alloy:"scrape_classic_histograms,attr,optional"`
+	// Whether to scrape native histograms.
+	ScrapeNativeHistograms bool `alloy:"scrape_native_histograms,attr,optional"`
```

Comment:
Related to this Prometheus change: https://github.com/prometheus/prometheus/pull/13987

We don't have to have the new `scrape_native_histograms` argument in order to maintain backwards compatibility with older Alloy versions. However, [the Prometheus docs](https://prometheus.io/docs/prometheus/latest/feature_flags/#native-histograms) mention that much of this functionality is experimental, so I thought it'd be good to be able to disable it.

By default `scrape_native_histograms` is set to `true` to maintain backwards compatibility.